quickstarts/microsoft.operationsmanagement/azure-sentinel/main.bicep

```diff
@@ -1,27 +1,33 @@
-param workspaceName string
 param location string = resourceGroup().location
+param sentinelName string = 'sentinel'
+param sku string = 'PerGB2018'
```

Comment:
are there allowed values we can add here?

Reply:
sentinelName can pretty much be whatever and I'd actually remove the sku parameter because only PerGB2018 works for Sentinel deployments

Reply:
sku was what I was referring to - removing it is fine (esp if no other value works at the moment ;))

Reply:
Yes, removed indeed!